docs/changes/0.46.0.rst

```diff
@@ -0,0 +1,106 @@
+QCoDeS 0.46.0 (2024-06-10)
+==========================
+
+Breaking Changes:
+-----------------
+
+
+- This will be the last version of QCoDeS to support Python 3.9
+
+- The keyword arguments expected to be passed to ``InstrumentBase`` and ``VisaInstrument`` subclasses are now
+  documented as TypedDics classes that can be used to type `**kwargs` in the subclass constructors.
+  See `Creating QCoDeS instrument drivers` for usage examples.
+
+  This also means that the these arguments **must** be passed as keyword arguments, and not as positional arguments.
```

Comment:
This also means that ~the~ these arguments **must** be passed as keyword arguments, and not as positional arguments.

Reply:
@einsmein This pr has been merged for 2 days. If you want to suggest changes please open a pr or an issue.

Reply:
Yep I didn't refresh the page before adding the comment. It's a small typo we can live with.